### ISSUE TITLE
Content corrupted errors are suddenly happening when searching.

### ISSUE THREAD

**Issue by wisniewskit** [wisniewskit/google-search-fixer#1]
Upon investigation, Google has apparently just enabled a [service worker](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API) for Chrome for Android 62 and up, and that service worker is not standards-compliant. It triggers the following error, which causes the searches to end up "corrupted":
```
Failed to load ‘https://www.google.ca/search?[snip].
A ServiceWorker passed a promise to FetchEvent.respondWith() that
rejected with ‘TypeError: HEAD or GET Request cannot have a body.’
```

Since my addon spoofs Firefox to act like the most recent Chrome version, the bug affects it. For the time-being, I will have to have the addon block the service worker.

Unfortunately, already-affected users will have to visit the URL `about:serviceworkers` and remove it manually from there (after upgrading, so it gets blocked instead of just reinstalling itself the next time a search is made).

**Comment by wisniewskit** [comment]
Fixed in commit 87cd99c13b4914c44d8af93132e652b92f61f46e.

**Comment by wisniewskit** [comment]
This was happening again today, as well as other issues with hanging searches, so I'm going to block the serviceworker again until I have some time to investigate what the deal is this time.
